### PR TITLE
[T4A3][T13-B3]Ong Qi Yong

### PR DESCRIPTION
**Pull request by oqyxxy**

### CODE REVIEW

**Comment by oqyxxy** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contact in the address book.
+ * Abstract class for Phone, Email and Address
+ */
+public abstract class Contact {
```
Comment (z11i) [inline src/seedu/addressbook/data/person/Contact.java:7]: :+1: use of abstract class, suitable for our use case.

File: src/seedu/addressbook/data/person/Email.java [hunk 2]
```diff
@@ -6,15 +6,15 @@
...
+public class Email extends Contact{
 
     public static final String EXAMPLE = "valid@e.mail";
     public static final String MESSAGE_EMAIL_CONSTRAINTS =
             "Person emails should be 2 alphanumeric/period strings separated by '@'";
     public static final String EMAIL_VALIDATION_REGEX = "[\\w\\.]+@[\\w\\.]+";
 
     public final String value;
-    private boolean isPrivate;
+    public boolean isPrivate;
```
Comment (z11i) [inline src/seedu/addressbook/data/person/Email.java:17]: Better for `isPrivate` to be `private`, and create a getter for it.

**Comment by z11i** [comment]
Generally a good job. Notice that it is not a good idea to expose your variables unless it's necessary.

**Pull request closed without merging**